### PR TITLE
app-doc/kicad-doc: Bump version to 4.0.2

### PR DESCRIPTION
**Pull request by zpuskas**
Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by zpuskas** [comment]
cc: @idella 
cc: @jlec 
cc: gentoo/proxy-maint

**Comment by a17r** [comment]
I'm sure these three PRs could be done within one, since they are closely related. As a benefit, github would view the diff between remove old and version bump.

**Comment by idella** [comment]
commit 1ab6ed627d0569096d32ebccaf8ca7c3857d2d73
Author: Zoltan Puskas <zoltan@sinustrom.info>
Date:   Sat Apr 16 03:10:49 2016 -0700

    app-doc/kicad-doc: Bump to version 4.0.2
    
    Package-Manager: portage-2.2.28
    
    Signed-off-by: Ian Delaney <idella4@gentoo.org>

**Comment by jlec** [comment]
@idella Please be much more careful in future when merging PRs as you are right now. Thanks.

File: app-doc/kicad-doc/kicad-doc-4.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (jlec) [inline app-doc/kicad-doc/kicad-doc-4.0.2.ebuild]: @zpuskas Wrong copyright year.

File: app-doc/kicad-doc/kicad-doc-4.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (jlec) [inline app-doc/kicad-doc/kicad-doc-4.0.2.ebuild]: Any reason why you don't use EAPI=6? Not using the latest EAPI creates more work on the long run.

File: app-doc/kicad-doc/kicad-doc-4.0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+FUSE="html pdf"
+LANGS="en fr it ja nl pl"
+LINGUAS=""
+for lang in ${LANGS} ; do
+	LINGUAS="${LINGUAS} linguas_${lang}"
+done
+IUSE="${FUSE} ${LINGUAS}"
```
Comment (jlec) [inline app-doc/kicad-doc/kicad-doc-4.0.2.ebuild]: Please unset ```lang```and ```LINGUAS``` after using them in global scope.

File: app-doc/kicad-doc/kicad-doc-4.0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	app-text/dblatex
+	>=app-text/po4a-0.45
+	>=sys-devel/gettext-0.18
+	dev-util/source-highlight
+	dev-perl/Unicode-LineBreak
+	linguas_ja? ( media-fonts/vlgothic )"
+RDEPEND=""
+
+src_prepare() {
+	DOCPATH="KICAD_DOC_INSTALL_PATH share/doc/kicad"
```
Comment (jlec) [inline app-doc/kicad-doc/kicad-doc-4.0.2.ebuild]: Could be local.

**Pull request closed without merging**